components/asset/introduction.rst

```diff
@@ -74,18 +75,27 @@ which defines the following two methods:
 :method:`Symfony\\Component\\Asset\\PackageInterface::getUrl`
     Returns an absolute or root-relative public path.
 
+With a package, you can:
+
+A) :ref:`version the assets <component-assets-versioning>`;
+B) set a :ref:`common base path (e.g. /css) <component-assets-path-package>`
```

Comment:
I would prefer moving `(e.g. /css)` outside the reference and put the directory name in a literal